Python/ibmcloudsql/SQLQuery.py

```diff
@@ -583,6 +583,8 @@ def export_job_history(self, cos_url=None, export_file_prefix = "job_export_", e
         url_parsed = self.ParsedUrl(self.export_cos_url)
 
         job_history_df = self.get_jobs() # Retrieve current job history (most recent 30 jobs)
+        job_history_df['error'] = job_history_df['error'].astype(unicode)
+        job_history_df['error_message'] = job_history_df['error_message'].astype(unicode)
```

Comment:
end_time too.
this should do it:
```
optional_string_cols = ['error', 'error_message', 'end_time' ]
job_history_df[optional_string_cols] = job_history_df[optional_string_cols].astype(unicode)
```
also, you may want to do the conversion already in get_jobs?